flixel/addons/effects/FlxEffectTrail.hx

```diff
@@ -0,0 +1,188 @@
+package flixel.addons.effects;
+
+import flixel.math.FlxMath;
+import flixel.math.FlxPoint;
+import flixel.util.FlxColor;
+import flixel.util.FlxDestroyUtil;
+import openfl.display.BitmapData;
+import openfl.geom.ColorTransform;
+import openfl.geom.Matrix;
+import openfl.geom.Point;
+using flixel.util.FlxArrayUtil;
+
+/**
+ * This creates a trail copying the bitmapData many times.
+ * 
+ * @author adrianulima
+ */
+class FlxEffectTrail implements IFlxEffect
```

Comment:
`FlxTrailArea` should be pretty light, it renders everything to a single Bitmap. Maybe it should be removed if they're similar enough..